test/test_matplotlib.py

```diff
@@ -28,3 +32,40 @@ def test_pdf(selenium):
     selenium.run("import io")
     selenium.run("fd = io.BytesIO()")
     selenium.run("plt.savefig(fd, format='pdf')")
+
+
+def test_rendering(selenium):
+    selenium.load_package("matplotlib")
+    selenium.run("""
+    from js import window
+    window.testing = True
+    from matplotlib import pyplot as plt
+    import numpy as np
+    t = np.arange(0.0, 2.0, 0.01)
+    s = 1 + np.sin(2 * np.pi * t)
+    plt.figure()
+    plt.plot(t, s, linewidth=1.0, marker=11)
+    plt.plot(t, t)
+    plt.grid(True)
+    plt.show()
+    """)
+
+    if not os.path.isfile('test/canvas-{0}.png'.format(selenium.browser)):
```

Comment:
Can you add a comment here: "If we don't have a reference image, write one to disk"